### PR TITLE
Add migration for new price type;

### PR DESCRIPTION
**Pull request by il12**
Change mongoose schema;
Change event type (add separate EventPrice type);
Change swagger schema;
ATTENTION: Remove migration from server.ts right after merge with prod

### CODE REVIEW

File: common/types/event.ts [hunk 1]
```diff
@@ -10,10 +10,10 @@ export type EventOnPoster = {
 	location: {
 		country: string;
 		city: string;
-        address: string;
+		address: string;
 	};
 	image: string;
-	price: string;
+	price: EventPrice;
```
Comment (bigKingPenguin) [inline common/types/event.ts:16]: EventPrice | null

File: common/types/event.ts [hunk 2]
```diff
@@ -41,3 +41,10 @@ export type SearchEventPayload = {
 	country?: string;
 	city?: string;
 };
+
+export type EventPrice = {
+	maxValue: number | null;
+	value: number | null;
+	minValue: number | null;
+	currency: number | null;
```
Comment (bigKingPenguin) [inline common/types/event.ts:49]: currency же string

Comment (il12) [reply common/types/event.ts]: Поправил

File: common/types/event.ts (hunk 1 above)
Comment (il12) [reply common/types/event.ts]: Поправил

**Review by bigKingPenguin — APPROVED** [review]

**Pull request merged**